src/translation/korean.c

```diff
@@ -104,6 +105,237 @@ static translation_string all_strings[] = {
     {TR_HOTKEY_SET_BOOKMARK_4, "4지점 설정"},
     {TR_HOTKEY_EDITOR_TOGGLE_BATTLE_INFO, "전투 정보 전환"},
     {TR_HOTKEY_EDIT_TITLE, "새 단축키 입력"}
```

Comment:
You're missing a `,` at the end here, make sure to add it so the code will compile.

Also, this time you got the PR right 😉

Reply:
I appreciate that. 
Only thing left here is fellow Korean translation contributor double-checking its text.